docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+    mongodb:
+        extends:
+            file: docker-compose.yml
+            service: mongodb
+        mem_limit: 4g
+        labels:
+            - triton.cns.services=mongodb
+        network_mode: bridge
```

Comment:
Might want to make a note to that effect... I'm sure someone will come along and deploy to swarm and be confuse by "local"